project/src/js/socket.js

```diff
@@ -20,13 +20,24 @@ let redrawOnDisconnectedTimeoutID;
 let proxyFailTimeoutID;
 const proxyFailMsg = 'The connection to lichess server has failed. If the problem is persistent this may be caused by proxy or network issues. In that case, we\'re sorry: lichess online features such as games, connected friends or challenges won\'t work.';
 
+const userPing = m.prop();
+const serverMoveTime = m.prop();
+
 const defaultHandlers = {
   following_onlines: handleFollowingOnline,
   following_enters: name => autoredraw(() => friendsApi.add(name)),
   following_leaves: name => autoredraw(() => friendsApi.remove(name)),
   challenges: data => {
     challengesApi.set(data);
     m.redraw();
+  },
+  mlat: mlat => {
```

Comment:
That can't be done like this. Since it will trigger a `redraw` on each ping event, that's a lot of useless redraws. The redraws should happend only when menu is open, and not all the time.